src/Type/Definition/EnumType.php

```diff
@@ -71,12 +71,10 @@ public function getValue($name)
         return $lookup[$name] ?? null;
     }
 
-    /**
-     * @return ArrayObject<string, EnumValueDefinition>
-     */
-    private function getNameLookup()
+    private function getNameLookup() : ArrayObject
     {
         if (! $this->nameLookup) {
+            /** @var ArrayObject<string, EnumValueDefinition> $lookup */
```

Comment:
and this can be maybe drop when return type doc is kept?

Reply:
I get fresh errors about conflict with `$this->nameLookup`... maybe easier to jusdt keep it for now. Furthermore, I had to remove the annotated return type, or the style checker complains...